web/profiles/joinup/config/install/search_api.index.collections.yml

```diff
@@ -0,0 +1,73 @@
+uuid: ac8e6731-1174-4f27-ab00-de719168b34b
+langcode: en
+status: true
+dependencies:
+  config:
+    - search_api.server.collections_search_server
+    - field.field.rdf_entity.collection.field_ar_description
+  module:
+    - search_api
+    - rdf_entity
+id: collections
+name: Collections
+description: ''
+read_only: false
+field_settings:
+  search_api_language:
+    label: 'Item language'
+    datasource_id: null
+    property_path: search_api_language
+    type: string
+    indexed_locked: true
+    type_locked: true
+  id:
+    label: ID
+    datasource_id: 'entity:rdf_entity'
+    property_path: id
+    type: string
+  label:
+    label: Title
+    datasource_id: 'entity:rdf_entity'
+    property_path: label
+    type: string
+  field_ar_description:
```

Comment:
Here is that weird field again, this seems to belong to a different entity or bundle. This should be `field_collection_description`.